src/ArcCommander/APIs/ConfigurationAPI.fs

```diff
@@ -173,4 +173,47 @@ module ConfigurationAPI =
         | false,false | false,true  -> 
             let workDir = GeneralConfiguration.getWorkDirectory arcConfiguration
             IniData.getLocalConfigPath workDir
-            |> unsetValueInIniPath
\ No newline at end of file
+            |> unsetValueInIniPath
+
+
+    /// Transfer git user metadata from arc config to git config.
+    let setGitUser (arcConfiguration : ArcConfiguration) (configurationArgs : Map<string,Argument>) =
+
+        let log = Logging.createLogger "ConfigurationSetGitUserLog"
+
+        log.Info("Start Config SetGitUser")
+
+        let nameOption = 
+            match tryGetFieldValueByName "Name" configurationArgs with
+            | Some name -> 
+                log.Trace("TRACE: Retrieved user name from given argument.")
```

Comment:
"TRACE: " prefix is not used. The only prefixes used are in `log.Warn` ("WARNING: "), `log.Error`, and `log.Fatal` ("ERROR: ")